examples/collection.json

```diff
@@ -1,42 +1,173 @@
+  ],
+  "item_assets": {
+    "cog": {
+      "title": "Mangrove cover",
+      "description": "Gridded estimate of mangrove cover",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "roles": [
+        "data"
+      ],
+      "classification:classes": [
+        {
+          "value": 0,
+          "name": "nodata",
+          "description": "nodata",
+          "nodata": true
+        },
+        {
+          "value": 1,
+          "name": "mangrove",
+          "description": "mangrove",
+          "color_hint": "009600"
+        }
+      ]
+    },
+    "change_cog": {
+      "title": "Mangrove cover change since 1996",
+      "description": "Gridded estimate of mangrove cover change since 1996",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "roles": [
+        "data"
+      ],
+      "classification:classes": [
+        {
+          "value": 0,
+          "name": "nodata",
+          "description": "nodata",
+          "nodata": true
+        },
+        {
+          "value": 1,
+          "name": "mangrove-gained",
+          "description": "mangrove gained",
+          "color_hint": "ff0000"
+        },
+        {
+          "value": 2,
+          "name": "mangrove-lost",
+          "description": "mangrove lost",
+          "color_hint": "0000ff"
+        }
+      ]
+    }
+  },
+  "version": "3.0",
+  "sci:doi": "10.5281/zenodo.6894273",
+  "sci:citation": "Bunting, P., Rosenqvist, A., Hilarides, L., Lucas, R., Thomas, N., Tadono, T., Worthington, T., Spalding, M., Murray, N., & Rebelo, L.-M. (2022). Global Mangrove Watch (1996 - 2020) Version 3.0 Dataset (3.0) [Data set]. Zenodo. 10.5281/zenodo.6894273",
+  "sci:publications": [
+    {
+      "doi": "10.3390/rs14153657",
+      "citation": "Bunting, P.; Rosenqvist, A.; Hilarides, L.; Lucas, R.M.; Thomas, T.; Tadono, T.; Worthington, T.A.; Spalding, M.; Murray, N.J.; Rebelo, L-M. Global Mangrove Extent Change 1996 \u2013 2020: Global Mangrove Watch Version 3.0. Remote Sensing. 2022"
+    }
+  ],
+  "renders": {
+    "mangroves": {
+      "assets": [
+        "cog"
+      ],
+      "colormap": {
+        "1": [
+          0,
+          150,
+          0
+        ]
+      }
+    },
+    "change": {
+      "assets": [
+        "change_cog"
+      ],
+      "colormap": {
+        "1": [
+          255,
+          0,
+          0
+        ],
+        "2": [
+          0,
+          0,
+          255
+        ]
+      }
+    }
+  },
+  "title": "Global Mangrove Watch (1996 - 2020) Version 3.0 Dataset",
   "extent": {
     "spatial": {
       "bbox": [
         [
-          -61.287624,
-          72.229798,
-          -52.301599,
-          74.622043
+          -82.0,
+          25.0,
+          -81.0,
+          26.0
         ]
       ]
     },
     "temporal": {
       "interval": [
         [
-          "2023-09-12T11:48:03.199913Z",
-          "2023-09-12T11:48:03.199913Z"
+          "2020-12-31T00:00:00Z",
```

Comment:
Might need to regenerate collection example

Reply:
Or I can just turn off the `collection.update_extent_from_items()` line!

Reply:
oh right, I forgot that was in the update-examples. I think that it would be fine to disable this line. Updating the extents seems like something more for ongoing collections rather than static collection examples